src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -87,6 +91,8 @@ public User findById(@ApiParam(value = "Id to find the user") @PathVariable Long
     })
     public User save(@RequestBody User user) {
 
+        user.setPassword(passwordEncoder.encode(user.getPassword()));
```

Comment:
The card requires that you have a different resource to update the password from a user, different from the resource where you are creating/updating them.